Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/Unit.cs

```diff
@@ -32,7 +33,18 @@ public class Unit {
 	}
 
 	public int GetStrength() {
-		return CurrentUpgrade == null ? BaseData.Strength : BaseData.Strength + CurrentUpgrade.Strength;
+		return CurrentUpgrade == null ? BaseData.Strength : BaseData.Strength + GetCombinedUpgradeStrength();
```

Comment:
this needs to return combined if currentTempUpgrade isn't null as well but currentUpgrade is.